### PR TITLE
Documentation README changes including architecture diagram and badges update

### PR DESCRIPTION
**Pull request by mtho11**
Some minor documentation updates to README.adoc:

1.  remove the POC content that is no longer relevant:
2. Update travis build badge and Bithound quality badges
3. Update architecture diagram to new repo:
miq-ui-architecture

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,12 +1,14 @@
-= Angular Components for ManageIQ
+= Angular UI Components for ManageIQ
 
-image::https://www.bithound.io/github/hawkular/hawkular-ui-components/badges/score.svg[link="https://www.bithound.io/github/hawkular/hawkular-ui-components"]
-image::https://www.bithound.io/github/hawkular/hawkular-ui-components/badges/dependencies.svg[link="https://www.bithound.io/github/hawkular/hawkular-ui-components/master/dependencies/npm"]
+image::https://www.bithound.io/github/ManageIQ/-ui-components/badges/score.svg[link="https://www.bithound
```
Comment (himdel) [inline README.adoc]: `ManageIQ/-ui-components` .. extra leading dash here?

Comment (mtho11) [inline README.adoc]: Yes, good catch @himdel

**Comment by himdel** [comment]
:+1: merged..

Btw, did you repaint the image or is it generated from some source? Might be worth it to add that source too..

**Comment by mtho11** [comment]
Unfortunately, I redrew the image from a proprietary drawing tool :(

**Comment by himdel** [comment]
Aah, ok, too bad then :(

**Pull request merged**